### DISCUSSION TITLE
Help needed for project setup for windows

### DISCUSSION THREAD

**Discussion by Shubhambawner** [primer/react#2284] [Q&A]
Has anyone tried to setup the project for development on a windows machine?
I did try with no luck, ran into issues, for more info [refer this thread](https://github.com/primer/react/issues/2250)
It would be very helpful if project doesn't require windows ppl to switch the OS, thanks in advance!

**Comment by lesliecdubs** [comment]
👋🏻 Hi @Shubhambawner, thanks for reporting this!

@pksjce since you're on first responder, do you mind taking a look at this and seeing what you can find out? I think you can add yourself to the BrowserStack entitlement if you need to test in a Windows environment. Thanks!
